packages/medusa-js/src/resources/admin/discounts.ts

```diff
@@ -133,6 +136,57 @@ class AdminDiscountsResource extends BaseResource {
     const path = `/admin/discounts/${id}/regions/${regionId}`
     return this.client.request("DELETE", path, {}, {}, customHeaders)
   }
+
+  /**
+   * @description creates a discount condition
+   */
+  createCondition(
+    id: string,
+    payload: AdminPostDiscountsConditions,
+    query: AdminPostDiscountsConditionsParams = {},
+    customHeaders: Record<string, any> = {}
+  ): ResponsePromise<AdminDiscountsRes> {
+    let path = `/admin/discounts/${id}/conditions`
+
+    if (query) {
+      const queryString = qs.stringify(query)
+      path = `/admin/discounts/${id}/conditions?${queryString}`
+    }
+
+    return this.client.request("POST", path, payload, {}, customHeaders)
+  }
+
+  /**
+   * @description Updates a discount condition
+   */
+  updateCondition(
+    id: string,
+    conditionId: string,
+    payload: AdminPostDiscountsConditionsCondition,
+    query: AdminPostDiscountsConditionsConditionParams = {},
+    customHeaders: Record<string, any> = {}
+  ): ResponsePromise<AdminDiscountsRes> {
+    let path = `/admin/discounts/${id}/conditions/${conditionId}`
+
+    if (query) {
+      const queryString = qs.stringify(query)
+      path = `/admin/discounts/${id}/conditions/${conditionId}?${queryString}`
+    }
+
+    return this.client.request("POST", path, payload, {}, customHeaders)
+  }
+
+  /**
+   * @description Removes a condition from a discount
+   */
+  deleteCondition(
+    id: string,
```

Comment:
same all the way 😂 sorry for the repetition. 
I let you update the react package too if we validate the naming

Reply:
np, renamed it in both `medusa-js` and `medusa-react`